helm/secrets-provider/tests/test-schema

```diff
@@ -216,10 +216,6 @@ function main() {
     conjur_authn_login_test ""
     update_results "$?" "$EXPECT_FAILURE"
 
-    announce "Missing Conjur authn login causes failure"
```

Comment:
For non-JWT functionality, the value `.Values.environment.conjur.authnLogin` is still required, and we want the user notified (as a "fail fast" mechanism) if this is missing, and it's not configured for authn-jwt.

We'll probably need to add an authnJWT.enabled boolean chart value to this Helm chart, and default it to "false". Then, either in the Helm chart `secrets-provider.yaml` template, or better yet, in the `values.schema.json`, we need to indicate that either `authnJWT.enabled` or `authnLogin` needs to be provided.

With the above changes, then the `missing_conjur_authn_login_test` should still be called here.

Reply:
I thought about `authnJWT.enabled` but because token projection is mandatory for authn-jwt then
`authnJWT.projectToken`.

Reply:
How can i do this test thing with values.schema.json?